src/core/eme/eme_manager.ts

```diff
@@ -170,7 +177,33 @@ export default function EMEManager(
                               value: initializationData });
       }
 
-      if (!handledInitData.storeIfNone(initializationData, true)) {
+      const lastKeyUpdate$ = new ReplaySubject<IKeyUpdateValue>(1);
+
+      // First, check that this initialization data is not already handled
+      const keyIds = initializationData.keyIds;
+      if (options.singleLicensePer === "content" &&
+          handledSessions.getLength() > 0 &&
+          keyIds !== undefined)
```

Comment:
Note: we may want to choose a strategy with unknown key-ids:
  - either we directly blacklist the corresponding initialization data
  - either we just do nothing (we will not be able to fallback from it).
 
Here we do even a third solution, which is to perform another license request, which I just realize make not much sense here. I might choose the second solution instead.

Reply:
Done